modules/structs/repo.go

```diff
@@ -94,6 +94,7 @@ type Repository struct {
 	HasPackages                   bool             `json:"has_packages"`
 	HasActions                    bool             `json:"has_actions"`
 	IgnoreWhitespaceConflicts     bool             `json:"ignore_whitespace_conflicts"`
+	ShowDependencies              bool             `json:"show_dependencies"`
```

Comment:
Ah, I think you misunderstood something:
We meant as a instance-wide setting, not a repo-wide setting.
Especially not changing the API, even though it isn't even shown there.

Reply:
that instance wide setting was my thirst thought.

But I also like the current aproach ... as each repo can enable it if they do need it, else it stays off

Reply:
Should I edit now to a instance-wide setting or leave it as it is?
Because I like the repo setting, so I can enable it for small to mid-sized repos and for really large repos where its slow I can disable it and I can change it without a restart of gitea
But maybe it can be a setting to enable/disable it on default?